### PR TITLE
openvpn: initial integration

### PR DESCRIPTION
**Pull request by DavidKorczynski**
@schwabe I have been doing some initial work on integrating OpenVPN into OSS-Fuzz to run fuzzers continuously on OpenVPN. I would be very happy to continue working on this and improving the integration as there are a lot more that can be done. If you are happy to integrate could you provide me with an email(s) that will receive bug reports? The email should be affiliated with a Google account.

The specific integration here is a first-step adoption of the work by @guidovranken described [here](https://guidovranken.com/2017/06/26/openvpn-fuzzers-released-notes/) - @guidovranken are you okay with me migrating your fuzzers?

@schwabe Let me know what you think as I would be very happy to get proper fuzzing integrated in OpenVPN.

### CODE REVIEW

**Comment by guidovranken** [comment]
Great!

**Comment by schwabe** [comment]
Yes that sounds like good idea. Everything that improves security is a good idea. You can take my arne@rfc2549.org email address that is the one that is affiliated with a Google account.

**Comment by DavidKorczynski** [comment]
This one is good to go @inferno-chromium @jonathanmetzman @asraa @oliverchang

File: projects/openvpn/build.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#
+################################################################################
+
+autoreconf -ivf
+./configure --disable-lz4
+make V=1
+cd src/openvpn
+rm openvpn.o
+ar r libopenvpn.a *.o
+for fuzzname in fuzz_base64 fuzz_dhcp fuzz_proxy fuzz_misc fuzz_buffer; do
```
Comment (oliverchang) [inline projects/openvpn/build.sh:24]: nit: do shell expansion here? i.e. `for fuzzname in fuzz*.cpp; ` then do `basename` to get rid of the extension where needed.

**Review by oliverchang — APPROVED** [review]

**Pull request merged**